### DISCUSSION TITLE
Publii synchronizes with wrong folder

### DISCUSSION THREAD

**Discussion by BertK19** [GetPublii/Publii#1702] [Help] [answered]
Hello dear members Publii,

As of several months my website didn't show updates after I sync them with Publii. After a long search I found out Publii syncs with another folder, not the one I appointed. I run my own Apache2 server here.
The remote path in Publii is:
/var/www/html/
It shows my website.
That is somehow changed into:
/home/musjepi/var/www/html/
For now I copy files to the right folder, which is a hassle.

I checked the Apache config files, all of them have: /var/www/html/

Is there a path inside Publii I can change? Filezilla blocks opening /home/musjepi/var/www/html/ 

Help is appreciated!

**Comment by bobmitro** [comment]
> Is there a path inside Publii I can change? Filezilla blocks opening /home/musjepi/var/www/html/

You can configure the remote path in the server settings, which is the directory where your website files will be stored on your server. This path should correspond to the correct directory on your server, such as public_html/, public_html/blog/

<img width="951" alt="screenshot 2024-09-19 at 13 19 33" src="https://github.com/user-attachments/assets/e3db41a6-62a4-4590-a5f9-edf116e7f7c0">

**Comment by BertK19** [comment]
Hello, I do have the right path setting. I'm on Linux and now I try to go back to Windows but the backup is not accepted. What do these errors mean?

![image](https://github.com/user-attachments/assets/2c63ea53-46d6-41df-918a-10693baa9601)

**Reply by dziudek** [reply]
@BertK19 -first of all please check if your backup contains the file mentioned in the error message input/config/site.config.json

**Reply by BertK19** [reply]
Yes the siteconfig.json is now in the input folder. I can sync but all files end up on the wrong directory.
Publii remote path is  /var/www/html/
but they are uploaded at the home directory, see below, look at the address bars

![IMG_20241003_161056](https://github.com/user-attachments/assets/15ba2469-5068-41dd-9e40-1e7d0ce30d66)

What I'm doing now for the time being is copy content from home directory to the site's html 
![IMG_20241003_160920](https://github.com/user-attachments/assets/f8912289-4ac0-4488-9298-daad8ff0988e)

**Comment by BertK19** [comment, marked as answer]
How I fixed this is: Instead of synchronizing I use WinSCP free file manager for uploading the contents of the output at html folder. It works!

- Filezilla could not enter the right folder. It is still a mystery to me.
- On the server part, I configured the output paths at the apach2 config files but it disconnected the website from internet.
